packages/fluentui/styles/src/types.ts

```diff
@@ -87,7 +87,7 @@ export interface FontFaceProps {
 
 export interface FontFace {
   name: string;
-  paths: string[];
+  paths?: string[];
```

Comment:
is this necessary ? an empty array works fine and is probably safer

Reply:
It seemed cleaner to make it optional. I'll close this PR in that case.